tests/test_base_device.py

```diff
@@ -115,14 +117,9 @@ def close(self):  # This is just to appease a test
         pass
 
 
-# pylint: disable=bare-except
-try:
-    service = QiskitRuntimeService(channel="ibm_quantum")
-    backend = service.backend("ibmq_qasm_simulator")
-except:
-    backend = MockedBackend()
-
+mocked_backend = MockedBackend()
 legacy_backend = MockedBackendLegacy()
```

Comment:
Yeah that one in my mind is a separate PR. I've been looking at `conftest.py`, and as a whole might it probably needs to be updated to use ibm_runtime_service since ibm_provider has officially been deprecated. With the cloud simulators being deprecated as well it's probably a good time to make some changes there.

Reply:
Awesome, sounds like a plan 🚀